package.json

```diff
@@ -7,6 +7,7 @@
     "firebase": "^9.1.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-icons": "^4.3.1",
```

Comment:
Nice icons! ⭐ 

Small details 😄 : I think it could be helpful please if you can update your PR description, and check ✔️ "Update dependencies" and "Testing steps" section, please 🙏  you might add `npm install` because you add new dependencies for the next mentors and teammates.

Reply:
Sure...